apps/davi/public/locales/en/translation.json

```diff
@@ -14,6 +14,21 @@
     "language": "Language",
     "languages": "Languages"
   },
+  "customRPC": {
+    "customRPC": "Custom RPC",
+    "customRPCUrl": "Custom RPC URL",
+    "enterURL": "Enter URL",
+    "useDefaultValue": "use default value",
+    "saveChanges": "Save Changes",
+    "genericError": "Seems to be a problem with the url provided",
+    "wrongChain": "The url provided is not from chain {{chainName}}",
+    "decentralizeMode": "Decentralize mode",
+    "decentralizeModeTooltip": "This will disable subgraph and only use RPCs. Some functions will be lost"
+  },
+  "decentralizeMode": {
+    "decentralizeMode": "Decentralize mode",
+    "decentralizeModeTooltip": "This will disable subgraph and only use RPCs. Some functions will be lost"
+  },

```

Comment:
We'll need to consider how we do translation work from now on. For now you could add the Spanish or make another task to translate the customRPC object